ls1intum/Artemis
# `Exam mode`: Fix alert when registering instructors or admins

### Checklist
#### General


#### Server
#### Client

### Motivation and Context


At the moment, it was not clear why there was an error in registering instructors or administrators for an exam.

### Description


- Added a new class: `AccessForbiddenAlertException`
- Caught the exception on the client and provide an alert


We should think about to refactor the `ErrorHandlerInterceptor` in the future to avoid duplicated or similar alerts, but for the moment this should be okay.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam

1. Log in to Artemis
2. Navigate to Exam Management
3. Try to register an instructor or admin
4. See the correct reason why action this is not successful

Please also verify that the change in `ConflictException.java` does not brake anything on the client :)

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


**Before:**
![image](https://user-images.githubusercontent.com/60005702/172586232-e0406d6c-c197-45d8-8dba-7137b7c6a506.png)

**After:**
![image](https://user-images.githubusercontent.com/60005702/172582801-8a17b55d-de9c-4253-b472-d353853e458e.png)
